.ci_support/linux_ppc64le_numpy2.0python3.10.____cpython.yaml

```diff
@@ -17,7 +17,7 @@ cxx_compiler:
 cxx_compiler_version:
 - '13'
 docker_image:
-- quay.io/condaforge/linux-anvil-ppc64le:alma9
```

Comment:
Why are these images changing?

Reply:
The images change to match the build architecture when using cross-compiling